lib/manageiq/messaging/kafka/queue.rb

```diff
@@ -2,6 +2,8 @@ module ManageIQ
   module Messaging
     module Kafka
       module Queue
+        GROUP_FOR_QUEUE_MESSAGES = 'manageiq_messaging_queue_group_'.freeze
```

Comment:
Not sure I like this being manageiq specific, but it's probably ok for now (maybe we should make this a parameter option somewhere?)